src/OrchardCore.Themes/TheAdmin/Assets/scss/modules/theme/darkmode/_bootswatch.scss

```diff
@@ -36,6 +37,10 @@
     }
 }
 
+.trumbowyg-fullscreen .trumbowyg-editor{
```

Comment:
Why we didn't introduce trumbowyg override to track all the WYSIWYG styles

Reply:
We don't need to really. It works fine as it is. Don't change what ain't broken they say 😉